### DISCUSSION TITLE
contribute software which isnt yours

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/winget-pkgs#15430] [Q&A] [answered]
hey i want to add the "razer synapse 2" software to the repository. But because of the "Contributer License" i dont know if I'm allowed to or not. and if i am allowed to add it, which of the 2 options i should choose.

- I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
- I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.

**Comment by GorujoCY** [comment, marked as answer]
second choice is what you should make if you are planning to fully submit programs that are not your own.

we hope to see you in an good effort of contribution :)

**Reply by ghost** [reply]
thank you :) 👍
